### PR TITLE
[ci] squashfs-tools: update to 4.5

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Topic Description
-----------------

squashfs-tools: update to 4.5

- Lint build scripts and texts.

Package(s) Affected
-------------------

`squashfs-tools`

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-admin/squashfs-tools/autobuild/build [hunk 1]
```diff
@@ -6,4 +6,4 @@ make XZ_SUPPORT=1 \
 
 abinfo "Installing {mk,un}squashfs ..."
 install -Dvm755 "$SRCDIR"/{mk,un}squashfs \
-    -t "$PKGDIR"/usr/bin/
```
Comment (MingcongBai) [inline extra-admin/squashfs-tools/autobuild/build:9]: What, why?

File: extra-admin/squashfs-tools/autobuild/defines [hunk 2]
```diff
@@ -1,4 +1,4 @@
 PKGNAME=squashfs-tools
 PKGSEC=utils
 PKGDEP="zlib lzo xz lz4"
-PKGDES="Tools for squashfs, a compressed read-only filesystem for Linux"
+PKGDES="Tools for Squashfs, a compressed read-only filesystem for Linux"
```
Comment (MingcongBai) [inline extra-admin/squashfs-tools/autobuild/defines]: SquashFS *

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-admin/squashfs-tools/autobuild/defines (hunk 2 above)
Comment (KexyBiscuit) [reply extra-admin/squashfs-tools/autobuild/defines]:
```suggestion
PKGDES="Tools for SquashFS, a compressed read-only filesystem for Linux"
```

**Comment by KexyBiscuit** [comment]
Ubuntu names the patches with CVE numbers, which looks better.
https://git.launchpad.net/ubuntu/+source/squashfs-tools/tree/debian/patches

**Review by MingcongBai — APPROVED** [review]

**Comment by KexyBiscuit** [comment]
Smoke test passed on Loongson 3.

**Pull request merged**